plugin/mocha.vim

```diff
@@ -63,7 +63,7 @@ function! s:GetNearestTest()
   let file = readfile(expand("%:p"))  "read current file
   let lineCount = 0                   "file line counter
   let lineDiff = 999                  "arbituary large number
-  let descPattern='\v\s*it\s*[(]?\s*([''"]{1})(.+)\1{1}'
+  let descPattern = '\v<(it|describe|context)\s*\(?\s*[''"](.*)[''"]\s*,\s*(function|-\>)'
```

Comment:
Oh, I didn't know that, I'm not familiar with ES6. Feel free to remove it then, the most important part was the '<' at the beginning, so it doesn't track transmit/emit/[...]